apps/scorecard/server/main.js

```diff
@@ -44,13 +44,20 @@ const doPlotParams = function () {
                 'these-days-of-the-week': ['Once'],
                 'these-days-of-the-month': ['Once'],
                 'these-months': ['Once'],
-                'dates': ['Recurring']
+                'scorecard-ends-on': ['Once'],
```

Comment:
This is a duplicate line. `'scorecard-ends-on'` is already the fourth line in this `hideOtherFor`